config/samples/clusters_v1beta1_cassandra.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
 spec:
   name: "username-Cassandra"
   version: "4.0.10"
+  privateNetworkCluster: true
```

Comment:
I think the defaul value here should be false

Reply:
Fixed